website/sidebars.js

```diff
@@ -93,16 +121,6 @@ const sidebars = {
         "tutorials/hello-tacos-tutorial"
       ],
     },
-    {
```

Comment:
Is the features section supposed to be removed? Also should the templates section be added to the sidebar at all if the doc is ready above?